app/templates/components/frost-modal-input.hbs

```diff
@@ -36,14 +36,15 @@
       {{frost-bunsen-form
         bunsenModel=formModel
         bunsenView=formView
+        disabled=formDisabled
         inline=true
         onChange=onChange
         onValidation=onValidation
         renderers=renderers
         showAllErrors=showAllErrors
         value=formValue
         validators=validators
-        }}
+      }}
       {{/if}}
```

Comment:
Darn, just realized this is not indented properly, guess the linting doesn't catch that :(